.gitignore

```diff
@@ -1,5 +1,7 @@
 .tox
 .hg*
+.venv*
+.idea
```

Comment:
- ``.idea`` is the project metadata folder of *PyCharm*, which is in pretty common use amongst our folks.
- When working with python virtualenvs, i think it is a convention to use ``.venv27`` or ``.venv34`` folders for designating the type of virtualenv. Given, when using *tox* one usually doesn't need manual-made virtualenvs for running the tests, but we usually even install *tox* into a project-local *virtualenv* to complete decouple from the system python.

Both entries shouldn't do any harm but instead should gain value for others, so they were added on purpose. However, we can remove them again if they make anyone sad.